src/assets/i18n/en.json

```diff
@@ -585,7 +585,9 @@
   "TRANSFER_STOPPED": "Transfer Cancelled",
   "TRY_AGAIN": "TRY AGAIN",
   "TRY_BEFORE_RATING": "Please try the content before rating it.",
-  "TRY_CONTENT_ON": "Try this content on the mobile app: ",
+  "SHARE_CONTENT_LINK": "See {{content_name}} on {{app_name}}: {{content_link}} \n Try this content on the mobile app: {{play_store_url}}",
+  "SHARE_APP_LINK": "Get {{app_name}} from the Play Store: {{play_store_url}}",
+  "SHARE_CONTENT_FILE": "{{app_name}}: {{content_name}} \n {{play_store_url}}",
```

Comment:
{{app_name}}: {{content_name}} \n
Try this content on the mobile app: {{play_store_url}}